mypy/expandtype.py

```diff
@@ -544,6 +543,8 @@ def remove_trivial(types: Iterable[Type]) -> list[Type]:
         * Remove everything else if there is an `object`
         * Remove strict duplicate types
     """
+    from mypy.state import state
```

Comment:
This nested imports also causes a small performance regression (maybe 0.1% to 0.2%).